front/src/hooks/stylo-export.js

```diff
@@ -28,3 +28,17 @@ export default function useStyloExport (bibStyle) {
 
   return { exportFormats, exportStyles, exportStylesPreview, isLoading }
 }
+
+export function useStyloExportPreview ({ md_content, bib_content, yaml_content }) {
+  const pandocExportEndpoint = useSelector(state => state.applicationConfig.pandocExportEndpoint)
+  const previewArgs = {
+    bibliography_style: 'chicagomodified',
```

Comment:
That parameter is optional and default to `chicagomodified` on the stylo export view but maybe you prefer to pass it explicitly?